android/src/main/java/com/urbanairship/reactnative/ReactAutopilot.java

```diff
@@ -140,28 +136,28 @@ public boolean onShowMessageCenter(@Nullable String messageId) {
         ReactNotificationProvider notificationProvider = new ReactNotificationProvider(context, airship.getAirshipConfigOptions());
         airship.getPushManager().setNotificationProvider(notificationProvider);
 
-        airship.shared().getAnalytics().registerSDKExtension(Analytics.EXTENSION_REACT_NATIVE, MODULE_VERSION);
+        airship.getAnalytics().registerSDKExtension(Analytics.EXTENSION_REACT_NATIVE, BuildConfig.MODULE_VERSION);
```

Comment:
airhsip.shared() is calling a class method on an instance to get the instance back, minor change